distribution/ddf-common/src/main/resources/bin/ddfsolr

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# Capture start|stop command
+# Set a default value in the event no arguments are provided
+COMMAND=""
+COMMAND=$1
+shift
+
+# In the secure configuration, Solr will only accept two-way TLS
+# connections. Additionally, Solr will reject any remote requests;
+# requests that do not originate from the local host.
+SECURE=true
+
+# -----------------------------------------------------------------------------
+usage() {
+    # -u Unsecure. Do not use TLS. Acceptremote requests
+    echo "Usage: $0 {start|stop} [-u]" >&2; exit 1;
+}
+
+while getopts ":u" o; do
+    case "${o}" in
+        u )
+            SECURE=false
+            ;;
+
+        * )
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+# -----------------------------------------------------------------------------
+if [ ! $COMMAND ]; then
+    echo "Missing command" >&2
+    usage
+fi
+
+if  ! ( [ "$COMMAND" == "start" ] || [ "$COMMAND" == "stop" ] ); then
+    echo "Unknown command: $COMMAND" >&2
+    usage
+fi
+
+SCRIPTDIR=$(dirname $0)
+HOME_DIR=$(cd "${SCRIPTDIR}/.."; pwd -P)
+GET=${SCRIPTDIR}/get_property
+SOLR_EXEC=${HOME_DIR}/solr/bin/solr
+SOLR_RC=0
+
+# Read port from system.properties file
+SOLR_PORT=$($GET "solr.http.port")
+
+# Do not proceed if required properties are missing
+if [ ! $SOLR_PORT ]; then
```

Comment:
Should change this to `if [ -z "$SOLR_PORT" ]; then` to catch both unset and empty string